docs/multicluster/getting-started.md

```diff
@@ -0,0 +1,424 @@
+# Antrea Multi-Cluster Installation
+
```

Comment:
You missed this comment?

> Better to have a short description of what Antrea Multi-cluster does. And it is supported since which version.

Reply:
done, Could you help to review again? I addressed most of them, there are one or two comments need your clarification. thanks.